R/0_R6_utils.R

```diff
@@ -51,7 +51,10 @@ printr <- function(..., file = nullfile(), sep = "", max_width = NULL) {
           ~ nchar(.) > max_width,
           ~ {
             break_locations <- stringr::str_locate_all(., " |$")[[1]][, 1]
-            split_width <- break_locations[max(which((break_locations - 1) < max_width))]
```

Comment:
Is there a reason to not replace most of this entire code block with `stringr::str_wrap`?

Reply:
No, that would be preferable. 
I just did not know / find that function